core/src/main/scala/org/apache/gearpump/cluster/ClusterMessage.scala

```diff
@@ -72,7 +73,9 @@ object AppMasterToMaster {
 
   case class GetAppData(appId: Int, key: String)
   case class GetAppDataResult(key: String, value: Any)
-  case class AppMasterDataDetail(appId: Int, application: Application)
+  case class AppMasterDataDetail(
+      appId: Int, appName: String = null, application: Application = null,
+      actorPath: String = null, clock: TimeStamp = 0, executors: List[String] = null)
 
```

Comment:
Should this class in the streaming package since the "clock" property belongs to streaming applications?